include/swift/SymbolGraphGen/SymbolGraphOptions.h

```diff
@@ -21,33 +21,33 @@ namespace symbolgraphgen {
 
 struct SymbolGraphOptions {
   /// The directory to output the symbol graph JSON files.
-  StringRef OutputDir;
+  StringRef OutputDir = StringRef{};
 
   /// The target of the module.
-  llvm::Triple Target; 
+  llvm::Triple Target = llvm::Triple{};
```

Comment:
These *are* default initialized already, but providing it doesn't hurt. You could just do ` = {};` if you want to keep though.

Reply:
I'm being extra paranoid now, since i was assuming that `bool` had a default initialization as well. 😅 I can clean up the initializer value, though. Thanks for the tip!